scripts/generate_docs.sc

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env amm
+
+/**
+  *
+  * DO NOT RUN THIS SCRIPT FROM THE SHELL!
```

Comment:
This comment makes no sense..   
first if not a shell then what?
second... if you give a warning like this why needs to be explained.